hr_attendance_rfid/security/ir.model.access.csv

```diff
@@ -1,4 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_hr_attendance_rfid,access.hr.attendance.rfid,hr_attendance.model_hr_attendance,hr_attendance_rfid.group_hr_attendance_rfid,1,1,1,0
-access_hr_employee_rfid,access.hr.employee.rfid,model_hr_employee,hr_attendance_rfid.group_hr_attendance_rfid,1,0,0,0
```

Comment:
Why removing this?

Reply:
Some things about it:

- `model_hr_employee` is not correct (try to install it and will get and error), maybe `hr.model_hr_employee` is correct (xml_id).
- IMO is not needed because now, apply `rfid_card_code` field to `hr.eemployee.base` and with these change, apply to `hr.employee` + `hr.employee.public `and users access to these models acording to access groups (all users access to `hr.employee.public` and users with employee groups, access to hr.employee)

What do you think about it?

Reply:
You still need access to that model for that group, as this is intended for having a user with just that permissions.

Reply:
Maybe you need to grant also access to hr.employee.public to this group?